CRM/Report/Form/Contribute/Summary.php

```diff
@@ -844,6 +854,7 @@ public function buildChart(&$rows) {
   public function alterDisplay(&$rows) {
     $entryFound = FALSE;
     $contributionStatus = CRM_Contribute_PseudoConstant::contributionStatus();
+    $batches = CRM_Batch_BAO_Batch::getBatches();
```

Comment:
Let's not load an array with all batches, as some orgs will create over a thousand a year, and yet the report will only use a very few.